torch/_inductor/freezing.py

```diff
@@ -77,9 +77,9 @@ def freeze(
     # See the details in fx_codegen_and_compile of compile_fx.py.
     view_to_reshape(aot_autograd_gm)
 
-    if torch._guards.TracingContext.get():
-        fw_metadata = torch._guards.TracingContext.get().fw_metadata
-        params_flat = torch._guards.TracingContext.get().params_flat
+    if tracing_context := torch._guards.TracingContext.get():
```

Comment:
walrus operator smelly -

Reply:
hm, is this a codebase convention or a personal preference? :p